tuf/conf.py

```diff
@@ -54,6 +54,10 @@
 # be saved to 'LOG_FILENAME'
 LOG_FILENAME = 'tuf.log'
 
+# This is the format/encoding of metadata that TUF will use. The options are
+# 'json' and 'der'. DER is an encoding of ASN.1 used for the Uptane project.
+METADATA_FORMAT = 'json' # 'der'
```

Comment:
We should verify with Justin that this is what we eventually want.  I also made this assumption with YAML, but only because it was simpler to implement.  If we eventually want to allow mixing of metadata formats, we'll have to modify the design a bit.